src/webview/devfile-registry/registryViewLoader.ts

```diff
@@ -132,18 +132,27 @@ export default class RegistryViewLoader {
 
 function getAllComponents(eventActionName: string, url?: string, error?: string) {
     let registries = ComponentTypesView.instance.getListOfRegistries();
-    if (url && url.length > 0) {
-        registries = registries.filter((registry: Registry) => registry.url === url);
-    }
-    const componentDescriptions = ComponentTypesView.instance.getCompDescriptions();
-    panel?.webview.postMessage(
-        {
-            action: eventActionName,
-            compDescriptions: Array.from(componentDescriptions),
-            registries: registries,
-            errorMessage: error
+    if (!registries) {
+        panel?.webview.postMessage(
+            {
+                action: eventActionName,
+                errorMessage: 'No Devfile registries configured'
```

Comment:
@msivasubramaniaan Instead of showing the Devfile Webview with this message, we can just load the Error Notification of VSCode saying `No devfile registry is accessible.`